hw/ip/kmac/data/kmac.hjson

```diff
@@ -0,0 +1,340 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+{ name: "kmac"
+  clock_primary: "clk_i"
+  bus_device: "tlul"
+  bus_host: "none"
+
+  interrupt_list: [
+    { name: "kmac_done"
+      desc: "KMAC/SHA3 absorbing has been completed"
+    }
+    { name: "fifo_empty"
+      desc: "Message FIFO empty condition"
+    }
+    { name: "kmac_err"
+      desc: "KMAC/SHA3 error occurred. ERR_CODE register shows the details"
+    }
+  ]
+  alert_list: []
+  param_list: [
+    { name:    "NumWordsKey"
+      type:    "int"
+      default: "16"
+      desc:    "Number of words for the secret key"
+      local:   "true"
+    }
+    { name: "NumWordsPrefix"
+      type: "int"
+      default: "11"
+      desc: "Number of words for Encoded NsPrefix."
+      local:"true"
+    }
+  ]
+  regwidth: "32"
+  registers: [
+    { name: "CFG"
+      desc: '''KMAC Configuration register.
+
+            This register is  updated when the hashing engine is in Idle.
+            If the software updates the register while the engine computes, the
+            updated value will be discarded.
+            '''
+      hwext:    "true"
```

Comment:
is this one supposed to be `hwext`?

Reply:
Ah... should be external but forgot to implement CFG register outside. The intention is to update CFG only when the engine is in idle.

Reply:
but if you wanted to update only when idle, you could use the same `regen` idea :)

Unless this config will also be set as part of the secret key case? Then it definitely makes sense to be outside.